packages/connect-popup/src/static/popup.html

```diff
@@ -1995,7 +1995,7 @@ <h3>New firmware update available</h3>
             <!-- Firmware not supported -->
             <div class="firmware-not-supported">
                 <div>
-                    <h3>Trezor One is not supported</h3>
+                    <h3>Coin is not supported</h3>
```

Comment:
this is a courageous change. Wouldn't it be better to text it something like
`Connected Trezor model not supported` 
?

Reply:
The string should never be used. However, makes sense, I will update it immediately